doc/source/user_guide/window.rst

```diff
@@ -198,6 +198,36 @@ from present information back to past information. This allows the rolling windo
 
    df
 
+.. _window.reverse_rolling_window:
+
+Reverse rolling window
+~~~~~~~~~~~~~~~~~~~~~~
+
+Get the window of a rolling function to look forward.
+
+Examples:
+
+.. ipython:: python
+
+   # Using slicing
+   # apply the rolling aggregation and then flip the result.
```

Comment:
looking at other example from this document, descriptions are typically outside the code blocks (rather than inside them as comments)

Reply:
Agreed. Would be better if this description was a paragraph before the example.

Reply:
done :+1: